src/cpu/minor/BaseMinorCPU.py

```diff
@@ -249,33 +269,6 @@ class MinorDefaultMiscFU(MinorFU):
     opLat = 1
 
 
-class MinorDefaultVecFU(MinorFU):
```

Comment:
The suite test is failing as the RISCV microbenchmark suite uses the riscvmatched board which uses these instructions https://github.com/gem5/gem5/blob/c890e6b113a59caeec3eea9bafea4a03a471f89d/src/python/gem5/prebuilt/riscvmatched/riscvmatched_core.py#L79

We can either just remove this from the riscvmatched board or we keep the vector instructions. I am not an expert at this so I am not sure what the best approach is.

Reply:
Thanks for taking a look, @Harshil2107. I believe we can remove that from the riscvmatched board since we are using the Simd types, and they are already in the MinorDefaultFloatSimdFU. I fixed the commit accordingly, but feedback from other folks would be appreciated.

Reply:
This is OK with me :)